### DISCUSSION TITLE
Additional button to switch on/off internal laptop screen

### DISCUSSION THREAD

**Discussion by oskpozn** [seerge/g-helper#1358] [Ideas]
Hey team,
Switching Hz in laptop screen is awesome feature, but I can't get out the feeling switching on/off the screen is missing in the panel.
There is an option in Windows to set laptop screen as off, but it's annoying to get in to the options and switch it there and it's not reliable when working on different workspaces.
I mostly work within a different desks with different set of peripherals and  I'm using the laptop screen as additional - switching it on only when it's needed. I guess there's a bunch of people treating their laptop as me, so the button would be a lot of help in managing workflow.  

Edit: 
Hence I was pointed out that Win + P option exist, closing the idea.

**Comment by seerge** [comment]
@oskpozn hello,

If you mean Windows Project settings (i.e. with external display connected), it can't be adjusted programmatically. You have to click there manually (or press Win+P) and then select desired mode.
